src/commandline_frame.ts

```diff
@@ -193,11 +199,16 @@ commandline_state.clInput.addEventListener(
             if (response.value.startsWith("ex.")) {
                 const [funcname, ...args] = response.value.slice(3).split(/\s+/)
 
-                if (args.length === 0) {
-                    commandline_state.fns[funcname]()
-                } else {
-                    commandline_state.fns[funcname](args.join(" "))
-                }
+                QUEUE[QUEUE.length - 1].then(() =>
```

Comment:
```suggestion
                QUEUE[QUEUE.length - 1].finally(() =>
```

Probably makes more sense, otherwise one failed command means that the command-line is unusable.